### ISSUE TITLE
Améliorer l'autoencodeur

### ISSUE THREAD

**Issue by PommeDev** [PommeDev/Detection-d-anomalies#10]
L'auto-encodeur actuel, est peu pertinent, voir même faux.
Donc il faut l'améliorer, pour générer des bonnes images, et pouvoir commencer la détection d'anomalie.

**Comment by josebadalmau** [comment]
Here are a few things you might want to try : 
- Use cuda to accelerate the training process (and run tests faster).
- Decrease the learning rate, it seems it is set to 0.1, which might be a bit high.
- Try and use the Binary Cross-Entropy loss instead of the MSE.
